cmd/ci-secret-bootstrap/main_test.go

```diff
@@ -106,23 +106,29 @@ const (
 secret_configs:
 - from:
     key-name-1:
-      bw_item: item-name-1
+      item: item-name-1
```

Comment:
Why is this conversation resolved? All the tests got changed

Reply:
I added below an item using the old fields as well. That should be enough, right?

Reply:
Please just leave the existing tests unmodified and add new ones for the new functionality.

Reply:
Got it. I'll make the changes. Thanks.